app/src/main/java/org/sopt/soptseminar_week1/api/RetrofitServiceCreator.kt

```diff
@@ -1,23 +1,26 @@
 package org.sopt.soptseminar_week1.api
 
+import com.google.gson.GsonBuilder
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitServiceCreator {
     private const val GITHUB_BASE_URL = "https://api.github.com"
-    private val retrofit_github: Retrofit =
-        Retrofit.Builder()
-            .baseUrl(GITHUB_BASE_URL)
-            .addConverterFactory(
-                GsonConverterFactory.create()
-            ).build()
-    val githubService: GithubService = retrofit_github.create(GithubService::class.java)
     private const val CHERISH_BASE_URL: String = "http://cherishserver.com"
-    private val retrofit_user: Retrofit =
-        Retrofit.Builder()
-            .baseUrl(CHERISH_BASE_URL)
+    private fun getRetrofit(baseUrl: String): Retrofit {
+        return Retrofit.Builder()
+            .baseUrl(baseUrl)
             .addConverterFactory(
-                GsonConverterFactory.create()
+                GsonConverterFactory.create(
+                    GsonBuilder()
+                        .create()
+                )
             ).build()
-    val userService: UserService = retrofit_user.create(UserService::class.java)
+    }
+
+    fun getGithubService(): GithubService =
+        getRetrofit(GITHUB_BASE_URL).create(GithubService::class.java)
+
+    fun getUserService(): UserService =
+        getRetrofit(CHERISH_BASE_URL).create(UserService::class.java)
```

Comment:
```suggestion
    fun getGithubService(): GithubService =
        getRetrofit(GITHUB_BASE_URL).create(GithubService::class.java)

// Sample.kt
getGithubService()
getGithubService()
getGithubService()
getGithubService()
getGithubService()
getGithubService()
getGithubService()
getGithubService()
```

이렇게 호출한 경우에는 메모리가 어떻게 될까?